cves/kernel/CVE-2018-12232.yml

```diff
@@ -106,21 +112,19 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 1da177e4c3f41524e886b7f1b8a0c1fc7321cac2
-  note: Discovered automatically by archeogit.
+  note: Manually Confirmed
 - commit: 86741ec25462e4c8cdce6df2f41ead05568c7d5e
-  note: Discovered automatically by archeogit.
+  note: Manually Confirmed
 - commit: c6d409cfd0fd41e7a0847875e4338ad648c9b96b
-  note: Discovered automatically by archeogit.
-- commit: c14ac9451c34832554db33386a4393be8bba3a7b
-  note: Discovered automatically by archeogit.
+  note: Manually Confirmed
```

Comment:
What are the differences between all these commits? How did they contribute to the vulnerability?